### PR TITLE
Add Preview page for adaptive sampling

### PR DESCRIPTION
**Pull request by bm1549**
Relates to this [DOCS](https://github.com/DataDog/documentation/pull/25397) ticket

### What does this PR do? What is the motivation?

Adds a new page (and links) that describe our new adaptive sampling feature.

🎉  This officially marks the transition to `Preview` for adaptive sampling

### Merge instructions

Looking for approval from either @yuanyuanzhao3 or @dussault-antoine prior to merge

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/tracing/guide/adaptive_sampling.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+Find below the minimum tracing library version required for the feature:
+
+| Language    | Minimum version required |
+|-------------|--------------------------|
+| Java        | [v1.34.0][5]             |
+| Go          | [v1.68.0][6]             |
+| Python      | [v.2.9.6][10]            |
+| Ruby        | [v2.0.0][11]             |
+| Node.js     | [v5.16.0][12]            |
+| .NET        | [v.2.54.0][13]           |
```
Comment (lucaspimentel) [inline content/en/tracing/guide/adaptive_sampling.md:49]: I was actually looking for this list today and remember you had pushed a docs PR, so I came to see if it was here. Thanks!

However, some of these values start with `v` and others with`v.`
```suggestion
| Python      | [v2.9.6][10]             |
| Ruby        | [v2.0.0][11]             |
| Node.js     | [v5.16.0][12]            |
| .NET        | [v2.54.0][13]            |
```

File: content/en/tracing/trace_pipeline/ingestion_controls.md [hunk 2]
```diff
@@ -128,7 +128,9 @@ See the **Datadog Agent and tracing library versions** your service is using. Co
 
 ### Configure the service ingestion rate
 
-<div class="alert alert-info"><strong>Remotely configured sampling rules are in Beta</strong>. Request access to the feature via this <a href="https://www.datadoghq.com/private-beta/resource-based-sampling-adaptive-sampling/">link</a> to be able to dynamically set this configuration from the Datadog UI without having to redeploy your service. Follow the instructions in the <a href="/tracing/guide/resource_based_sampling">Resource-based sampling guide</a> to get started.</div>
+<div class="alert alert-info"><strong>Adaptive sampling rates are in Preview</strong>. Request access to the feature via this <a href="https://www.datadoghq.com/private-beta/resource-based-sampling-adaptive-sampling/">link</a> to have Datadog adapt to your traffic to keep you under your configured monthly ingestion target.e. Follow the instructions in the <a href="/tracing/guide/adaptive_sampling">Adaptive sampling guide</a> to get started.</div>
```
Comment (lucaspimentel) [inline content/en/tracing/trace_pipeline/ingestion_controls.md:131]:
```suggestion
<div class="alert alert-info"><strong>Adaptive sampling rates are in Preview</strong>. Request access to the feature via this <a href="https://www.datadoghq.com/private-beta/resource-based-sampling-adaptive-sampling/">link</a> to have Datadog adapt to your traffic to keep you under your configured monthly ingestion target. Follow the instructions in the <a href="/tracing/guide/adaptive_sampling">Adaptive sampling guide</a> to get started.</div>
```

**Review by brett0000FF — CHANGES_REQUESTED** [review]
Thanks for writing up this content! I left a few suggestions for you to consider. The main blocker for me approving this is the images. Seems like they weren't captured in the Demo org, and some of them include usernames. Let me know once you've reviewed my suggestions and I can take another look!

File: content/en/tracing/guide/adaptive_sampling.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
+---
+title: Adaptive sampling
+
+disable_toc: false
+private: true
```
[the remark this answers is not in the document]
Comment (bm1549) [reply content/en/tracing/guide/adaptive_sampling.md:5]: I was following the existing precedent set for [resource_based_sampling](https://github.com/DataDog/documentation/blob/master/content/en/tracing/guide/resource_based_sampling.md)

@dussault-antoine any reason to keep this private? If not, I can remove these lines

File: content/en/tracing/guide/adaptive_sampling.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  - `Automatic` if the [default head-based sampling mechanism][8] from the Agent applies.
+  - `Local Configured` if a [sampling rule][7] was set locally in the tracing library.
+  - `Remote Configured` if a remote sampling rule was set from the Datadog UI. To learn how to configure sampling rules from the Ingestion Control page, read the section on [remotely configuring sampling rules][15].
+  - `Adaptive` if the sampling rules are set remotely by Datadog.
+
+## Remotely configure sampling rules for the service
+
+To configure adaptive sampling for the service:
+1. Click **Manage Ingestion rate**. If the remote configuration option is disabled, make sure that the listed [requirements](#compatibility-requirements) are all met.
+   {{< img src="/tracing/guide/adaptive_sampling/ingestion_control_page.png" alt="Ingestion Control Page" style="width:100%;">}}
```
[the remark this answers is not in the document]
Comment (bm1549) [reply content/en/tracing/guide/adaptive_sampling.md]: Removed - we can always add them back in if we decide we need them

That being said, I don't have a demo account so I'll need a bit of help there

File: content/en/tracing/guide/adaptive_sampling.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+1. Configure a **Monthly Ingestion Target** for adaptive sampling.
+   {{< img src="/tracing/guide/adaptive_sampling/monthly_ingestion_target.png" alt="Monthly Ingestion Target" style="width:100%;">}}
+1. For your service, go to **Manage Ingestion Rate**
+   {{< img src="/tracing/guide/adaptive_sampling/enroll_service.png" alt="Enroll Service to Adaptive Sampling" style="width:100%;">}}
+1. Set your service's sampling strategy to **Datadog Adaptive Sampling Rates** and click **Apply** to save the configuration.
+   {{< img src="/tracing/guide/adaptive_sampling/adaptive_sampling_rate.png" alt="Datadog Adaptive Sampling Rates" style="width:100%;">}}
```
[the remark this answers is not in the document]
Comment (bm1549) [reply content/en/tracing/guide/adaptive_sampling.md]: Removed

**Review by brett0000FF — APPROVED** [review]
🚀

**Comment by yuanyuanzhao3** [comment]
This looks good to me. I have a few small changes on the branch. Just a heads up as I need to resolve conflict before I can push.

**Review by dussault-antoine — APPROVED** [review]

**Comment by dussault-antoine** [comment]
Good to go on my side

**Pull request merged**